beta-src/src/components/ui/WDUI.tsx

```diff
@@ -50,10 +57,9 @@ const WDUI: React.FC = function (): React.ReactElement {
     user,
     year,
   } = useAppSelector(gameOverview);
-  const newMessagesFrom = useAppSelector(
-    ({ game }) => game.messages.newMessagesFrom,
-  );
 
+  const { messages } = useAppSelector(gameMessages);
```

Comment:
I don't think this is a great pattern for react. If you need to destructure out only a single field from what you get from useAppSelector, it's better to write a more specific selector that gives you only that single field.

As it stands, even if messages doesn't change, this selector I think will cause a rerender on *every* new query to get messages, because the GameMessages object also has within it the time field, which does keep updating.